### ISSUE TITLE
Matlab documentation updates

### ISSUE THREAD

**Issue by craig8** [VOLTTRON/volttron#2099]
NOTE issues are for bugs/feature requests not support.  Please ask at https://stackoverflow.com/questions/tagged/volttron for help

**Comment by craig8** [comment]
@schandrika I disagree with the don't start the agent....the point is to use the config store of the agent to trigger the publish...not the starting of the agent.

**Comment by schandrika** [comment]
In that case we should at least install and start without a config file.

**Comment by craig8** [comment]
@schandrika Yes installing without a config file would be fine... @sgilbride  you can pass {} as the --config parameter so that it won't be required

**Comment by craig8** [comment]
@sgilbride please verify this is complete and updated per our latest experience with new users.
